gensim/corpora/lowcorpus.py

```diff
@@ -119,7 +119,7 @@ def line2doc(self, line):
                     marker.add(word)
             # construct a list of (wordIndex, wordFrequency) 2-tuples
             doc = list(zip(map(self.word2id.get, use_words),
-                      map(words.count, use_words)))
+                           map(words.count, use_words)))
```

Comment:
No vertical indent in gensim; use hanging indent in all cases (& no statement on first line).